s3_downloader.go

```diff
@@ -116,7 +116,7 @@ func (downloader s3Downloader) RemoteChecksum(remotePath string) (string, error)
 	defer os.RemoveAll(dir)
 	hashFile := filepath.Join(dir, "md5Hash")
 
-	err = downloader.Download(hashFile, hashRemotePath)
+	err = downloader.Download(hashRemotePath, hashFile)
```

Comment:
🤦

Reply:
This should be covered in the test, no?

Reply:
Yes, but how? We can't easily spin up a minio/S3 instance in our current test setup

Reply:
I'm up for suggestsions!

Reply:
Yeah, we could wire in a mock for the s3/manager bit... don't think it seems worth it though. /shrug